src/sciphi_config.h

```diff
@@ -169,12 +169,15 @@ class Config{
 	void setTmpParam(ParamType param, double newParam);
```

Comment:
`src/sciphi-findBestTrees.o` binary committed

Reply:
... and gone again.